src/udev/udev-builtin-net_setup_link.c

```diff
@@ -48,6 +50,12 @@ static int builtin_net_setup_link(sd_device *dev, sd_netlink **rtnl, int argc, c
         if (link->new_name)
                 udev_builtin_add_property(dev, test, "ID_NET_NAME", link->new_name);
 
+        joined = strv_join(link->config->dropins, NULL);
```

Comment:
As you commented, a dropins may contains space in its name. I think it is better to escape spaces in the name.

Reply:
Do we have any existing helper to do this?

Reply:
I do not think so.

How about something like the following (no error handling included below)?
```c
char **escaped = new0(char*, strv_length(link->config->dropins));
size_t i = 0;
STRV_FOREACH(s, link->config->dropins)
        escaped[i++] = strreplace(s, " ", "\\ ");
```

Reply:
We now use ":" as the separator and escape/unescape all ":" in the pathname